rfc/website-revamp.md

```diff
@@ -0,0 +1,143 @@
+# Website Revamp
+
+| | |
+|-|-|
+| **Status**     | Proposed|
+| **RFC #**      | [#110](https://github.com/spinnaker/governance/pull/110) |
+| **Author(s)**  | Aimee Ukasick ([@aimeeu](https://github.com/aimeeu)), Brian Le ([@brian-armory](https://github.com/brian-armory)), Rosalind Benoit ([@dnilasor](https://github.com/dnilasor)) |
+| **SIG / WG**   | Documentation SIG |
+
+## Overview
+
+Spinnaker.io is the public face of the Spinnaker project. This RFC proposes several improvements to .io and a pathway to achieving them.
+
+## Goals and Non-Goals
+
+### Goals
+
+The ultimate goal is a more modern looking and consistently styled spinnaker.io.
+
+To that end, these are the specific goals we are working toward:
+
+* Provide clear entryways and paths for the different personas spinnaker.io should speak to. For example, a more clear and opinionated navigation for someone brand new to Spinnaker.
+* Provide a clear and compelling answer to anyone asking, “Why Spinnaker for Continuous Delivery?”
+* Add useful documentation features, e.g. versioned documentation and a better in-page table of contents
+* Improve operator, end-user, and contributor experience by providing more navigable documentation.
+* Improve content organization to facilitate easy contribution.
+* Improve local build time and adopt the new way GitHub pages uses themes
+
+### Non-Goals
+
+* Not a complete reorganization of the documentation set. What will happen is the current documentation related tabs will be condensed under a single “Docs” tab in the top navigation.
+* Not replatforming or rethemeing .io. The cost/benefit is not there to do this at this time.
+
+## Motivation and Rationale
+
+Spinnaker.io has operated with one basic design look/feel since March 22, 2017. Given the current momentum and size of the project, it is time for a refresh. The Docs SIG feels it’s time to evolve .io beyond a documentation site and into a project hub, providing a presence for Spinnaker that is clearly segmented for different user personas.
+
+We need to provide context to attract new companies and ICs as they encounter Spinnaker. This aligns with Travis Tomsu’s efforts to add more context and persuasive oomph to READMEs across the project. The window for Spinnaker inevitability is threatened by a crowded marketplace and will close; this project will boost Spinnaker’s profile and chances for inevitability.
+
+From a technical standpoint, the way GitHub pages manages themes has changed to remote themes, and our implementation is outdated. Relying on the Minimal Mistakes theme-maintainer’s repository results in less theme-related maintenance for the Docs SIG and capture of theme feature improvements.
+
+## Timeline
+* Achieve buy-in from stakeholders by March.
+* Finalize Statement of Work and share with community in March.
+* Finalize SIG-validated high-level design in April.
+* Sitemap
+* Accept contractor bids, including billing rate and time estimate, in April.
```

Comment:
Has there been discussion around what this contractor process would look like, and where the funds will come from to pay for it?  I don't know of any precedent for this in the Spinnaker project, so would like to make sure this is fully vetted by the TOC/SC and probably also the CDF before we move forward.

Reply:
We discussed this in the TOC/SIG leads sync. The conclusion we came to was that @plumpy is going to help @dnilasor ask the CDf for $$$$ once this RFC is approved.

Reply:
@ezimanyi @brian-armory I synced with @plumpy on this and am reaching out to Dan Lopez about it today to start the conversation as it may take a while to fully capture his attention. Agree it should be vetted, and if possible, sponsored by the CDF.